Mathlib/MeasureTheory/Function/LpSpace/Basic.lean

```diff
@@ -445,7 +445,6 @@ instance instIsScalarTower [SMul 𝕜 𝕜'] [IsScalarTower 𝕜 𝕜' E] : IsSc
   smul_assoc k k' f := Subtype.ext <| smul_assoc k k' (f : α →ₘ[μ] E)
 
 instance instIsBoundedSMul [Fact (1 ≤ p)] : IsBoundedSMul 𝕜 (Lp E p μ) :=
-  -- TODO: add `IsBoundedSMul.of_nnnorm_smul_le`
```

Comment:
This still applies, but for `enorm_smul_le`, right?

Reply:
Fair point; I added the enorm version :-)

Reply:
Isn't the point to use the new definition here?

Reply:
Sure; I have golfed the proof now